# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libcsv/makefile.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- /dev/null	1970-01-01 01:00:00.000000000 +0100
+++ ./Makefile	2006-10-18 18:32:18.000000000 +0200
@@ -0,0 +1,35 @@
+CC=gcc
+AR=ar
+
+DESTDIR=
+LIBDIR=/usr/lib
+INCDIR=/usr/include
+
+.PHONY: all install install_headers install_shared install_static
+
+all: libcsv.so libcsv.a
+
+install: install_headers install_shared install_static
+
+install_headers: csv.h
+	mkdir -p $(DESTDIR)$(INCDIR)/libcsv/
+	cp -f $^ $(DESTDIR)$(INCDIR)/libcsv/
+
+install_shared: libcsv.so
+	cp -f $< $(DESTDIR)$(LIBDIR)/$<.0.9.0
+	ln -sf $<.0.9.0 $(DESTDIR)$(LIBDIR)/$<.0
+	ln -sf $<.0 $(DESTDIR)$(LIBDIR)/$<
+
+install_static: libcsv.a
+	cp -f $< $(DESTDIR)$(LIBDIR)/$<
+
+libcsv.o: libcsv.c csv.h
+	$(CC) $< -c -o $@
+
+libcsv.so: libcsv.o
+	$(CC) -shared $< -o $@
+
+libcsv.a: libcsv.o
+	$(AR) -rc $@ $^
+	$(AR) -s $@
+
